src/gpodder/plugins/itunes.py

```diff
@@ -69,6 +85,39 @@ def __init__(self):
         self.priority = directory.Provider.PRIORITY_SECONDARY_SEARCH
 
     def on_search(self, query):
-        json_url = 'https://itunes.apple.com/search?media=podcast&term={}'.format(urllib.parse.quote(query))
-
-        return [directory.DirectoryEntry(entry['collectionName'], entry['feedUrl'], entry['artworkUrl100']) for entry in util.read_json(json_url)['results']]
+        offset = 0
+
+        while True:
+            json_url = (
+                f'https://itunes.apple.com/search?media=podcast&term='
+                f'{urllib.parse.quote(query)}&limit={PAGE_SIZE}&offset='
+                f'{offset}'
```

Comment:
IMHO it would be fine if this wasn't split into 3 lines, and just a single f-string on one line.

Reply:
That would greatly exceed 80/100 column line length though?

I'm merging soon as is (after adding the stop condition comment back in) since this PR should never have sat as long as it did (my fault), stuff can always be changed again later.